modules/p2/ding_entity_rating/ding_entity_rating.module

```diff
@@ -345,14 +345,54 @@ function ding_entity_rating_build_theme_list($rating) {
  *   The average rating of the given entity.
  */
 function ding_entity_rating_get_average($entity) {
-  $query = db_select('field_data_field_weight', 'w');
-  $query->join('eck_ding_type', 'dt', 'w.entity_id = dt.id');
-  $query->join('field_data_field_value', 'v', 'w.entity_id = v.entity_id');
-  $query->condition('v.field_value_value', $entity->ding_entity_id)
-    ->addExpression('AVG(w.field_weight_value)', 'avg_count');
-  $result = $query->execute();
-
-  return $result->fetchField(0);
+  return ding_entity_rating_average($entity);
+}
+
+/**
+ * Set the average rating of a ting object entity.
+ *
+ * @param object $entity
+ *   The ting object entity to remove from the rating list.
```

Comment:
Nitpicking here but this bit of DocBlock seems copy/pasted. Could you please update it?